src/QuartoGameState.java

```diff
@@ -4,9 +4,9 @@
 import java.util.Iterator;
 
 
-public class QuartoGameState implements Iterable<QuartoGameTransition> {
+public class QuartoGameState {
 
-    private static HashMap<String, QuartoGameState> registeredStates = new HashMap<String, QuartoGameState>();
+    public static HashMap<String, QuartoGameState> registeredStates = new HashMap<String, QuartoGameState>();
```

Comment:
This should remain private, later when we thread we're going to need to access this through threadsafe methods